src/server/resolvers/index.ts

```diff
@@ -30,6 +31,7 @@ const userResolvers: Omit<UserResolvers, '__resolveType' | 'userType'> = {
 		return dietaryRestrictions.map(toEnum(DietaryRestriction));
 	},
 	email: async user => (await user).email,
+	eventsAttended: async user => (await user).eventsAttended || [],
```

Comment:
Update this if you change eventsAttended to be optional.

Reply:
Updated; || removed